src/Symfony/Component/Mailer/README.md

```diff
@@ -3,9 +3,66 @@ Mailer Component
 
 The Mailer component helps sending emails.
 
+Getting Started
+---------------
+
+```
+$ composer require symfony/mailer
+```
+
+```php
+use Symfony\Component\Mailer\Transport;
+use Symfony\Component\Mailer\Mailer;
+
+$transport = Transport::fromDsn('sendgrid://KEY@default');
```

Comment:
`sendgrid+smtp`?

Reply:
According to https://symfony.com/doc/current/mailer.html#using-a-3rd-party-transport , `sendgrid://` will choose the best protocol (and `+smtp` allows you to force a protocol).

Reply:
I would use an SMTP server as an example here instead.